Automattic/wp-calypso
# Remove survey from jetpack agency dashboard

## Proposed Changes

* Remove survey from jetpack agency dashboard

## Testing Instructions

1. Checkout this branch
2. Start Jetpack Cloud via yarn start-jetpack-cloud
3. If you are not already marked as an agency, you can mark yourself with these instructions: PCYsg-KIj-p2
4. Go to the Pro Dashboard at http://jetpack.cloud.localhost:3000/dashboard
5. Make sure the banner does not show up. (If you see the welcome banner, click on the "Got it" CTA and make sure the survey banner does not show up after it)
![image](https://github.com/Automattic/wp-calypso/assets/65001528/c2a33271-340a-4718-bfa6-a4b2cd82ec65)

## Pre-merge Checklist
